MiniApp/Classes/admob7/AdMobDisplayer.swift

```diff
@@ -128,6 +128,7 @@ extension AdMobDisplayer: GADInterstitialDelegate {
     public func interstitial(_ ad: GADInterstitial, didFailToReceiveAdWithError error: GADRequestError) { // swiftlint:disable:this identifier_name
         if let id = ad.adUnitID {
             onInterstitialLoaded[id]?(.failure(error))
+            cleanInterstitial(id)
```

Comment:
on AdMob 7 only interstitial needs to be cleaned, because google managed differently interstital and rewards loading. This was fixed when they released v8.0.0